### PR TITLE
Added line-clamp features to trim text

### PR DESCRIPTION
**Pull request by koskinpark**
frontend trim of text paragraphs pretty common issue. 
seems reasonable to use -webkit-line-clamp in browsers which support this (~ 85%)
https://caniuse.com/#feat=css-line-clamp
and have failback in js for browsers which not support.

this PR allows to set config for all lines which should be trimmed. with various amount of lines based on theme breakpoints.

### CODE REVIEW

File: src/sass/_init.scss [hunk 1]
```diff
@@ -2,3 +2,6 @@
 @import 'abstractions/colors';
 @import 'abstractions/variables';
 @import '~susy';
+
+// Uncomment next @import if you need trimming feature on the project.
+//@import 'line-clamp';
```
Comment (Megajack) [inline src/sass/_init.scss]: if we don't need this feature for all projects, maybe have sense to make it as component and move outside the theme? somewhere to Skilld component library, and load it only when we need it

Comment (koskinpark) [reply src/sass/_init.scss]: why? it's disabled anyway

Comment (Megajack) [reply src/sass/_init.scss]: why we need code that we don't need/use? it's more logical to import it like component or library.

File: src/js/trimlines/trimlines.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ *
+ * If browser supports `-webkit-clamp-lines` will be used trimCSS, else trimJs.
+ */
+
+import lineClamp from 'line-clamp';
+import modernizr from '../.modernizrrc';
+import trimmedElements from './line-clamps';
+import {addBreakpointListener} from '../breakpoints';
+
+(function (Drupal, lineClamp) {
```
Comment (iberdinsky-skilld) [inline src/js/trimlines/trimlines.js]: what for drupal here?

**Comment by iberdinsky-skilld** [comment]
we need to update this with last changes of project

**Comment by iberdinsky-skilld** [comment]
this PR should be moved to separated package

**Comment by hog-seruj** [comment]
What about NPM package for this feature as well?

**Comment by hog-seruj** [comment]
https://github.com/hog-seruj/line-clamp

**Comment by hog-seruj** [comment]
https://www.npmjs.com/package/@skilld/line-clamp

**Pull request closed without merging**